### PR TITLE
openstack-cleanvm: Run job for Newton

### PR DESCRIPTION
**Pull request by toabctl**
cleanvm needs to run for Newton. Do this for SLE_12_SP2, Leap_42.1
and Leap_42.2 .

### CODE REVIEW

**Comment by bmwiedemann** [comment]
+1

File: scripts/jenkins/jobs-obs/openstack-cleanvm.yaml [hunk 1]
```diff
@@ -39,6 +40,7 @@
         && ( openstack_project=="Juno" && (image=="SLE_11_SP3"))
         || ( openstack_project=="Liberty" && (image=="SLE_12_SP1" || image=="SLE_12" || image=="openSUSE-Leap-42.1"))
         || ( openstack_project=="Mitaka" && (image=="SLE_12_SP1" || image=="SLE_12_SP2" || image=="openSUSE-Leap-42.1"))
+        || ( openstack_project=="Mitaka" && (image=="SLE_12_SP2" || image=="openSUSE-Leap-42.1" || image=="openSUSE-Leap-42.2"))
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-obs/openstack-cleanvm.yaml]: that should read ```Newton``` instead of ```Mitaka```

Comment (dirkmueller) [inline scripts/jenkins/jobs-obs/openstack-cleanvm.yaml]: and maybe only test Leap 42.2 instead of also leap 42.1 (which has been broken for ages)?

Comment (toabctl) [inline scripts/jenkins/jobs-obs/openstack-cleanvm.yaml]: Both done

**Comment by kbaikov** [comment]
+1

**Comment by dirkmueller** [comment]
+1

**Pull request merged**